src/feature-toggles/FeatureTogglesContext.tsx

```diff
@@ -0,0 +1,68 @@
+import React, {createContext, useContext, useEffect, useState} from 'react';
+import {storage} from '@atb/storage';
+import {useRemoteConfig} from '@atb/RemoteConfigContext';
+import {
+  DebugOverride,
+  FeatureToggles,
+  OverridesMap,
+  SetDebugOverride,
+} from './types';
+import {
+  getDebugOverridesFromSpecs,
+  getFeatureTogglesFromSpecs,
+  getStoredOverrides,
+} from './utils.ts';
+
+type FeatureTogglesContextState = FeatureToggles & {
+  debug: {
+    overrides: DebugOverride[];
+    setOverride: SetDebugOverride;
+  };
```

Comment:
Good input, always nice to think things through. As you mention, we have had problem with the debug overrides before since they were loaded when the components were mounted, and even if the fetch form async storage only took 0.1ms it was enough so that we got one render cycle with the wrong value.

I don't think this should be a problem anymore since now the overrides from async storage are loaded while the loading boundary shows, and since the boundary shows for minimum 1 second I think it should be more than enough time for loading them.